src/components/admin/Billing/index.tsx

```diff
@@ -35,25 +42,43 @@ function AdminBilling() {
     const setHydrated = useBilling_setHydrated();
     const setHydrationErrorsExist = useBilling_setHydrationErrorsExist();
 
+    const selectedTenant = useBilling_selectedTenant();
     const setBillingHistory = useBilling_setBillingHistory();
+    const updateBillingHistory = useBilling_updateBillingHistory();
     const setDataByTaskGraphDetails = useBilling_setDataByTaskGraphDetails();
 
     const resetBillingState = useBilling_resetState();
 
+    const [historyInitialized, setHistoryInitialized] =
+        useState<boolean>(false);
```

Comment:
This does not need typed

Reply:
It is widely understood that the type can be inferred. Adding a type annotation to all `useState` hooks is a personal preference for reasons that I do not care to explain. If you are adamant that you would like it removed from your codebase, I shall do so.